### PR TITLE
Update tool-windows.tmpl

### PR DESCRIPTION
**Pull request by bjornregnell**
changed "%1%" and %2% to "%~1" and %~2 to allow spaces in paths by surrounding quotes
according to advice at 
http://stackoverflow.com/questions/473117/pass-path-with-spaces-as-parameter-to-bat-file
http://ss64.com/nt/syntax-args.html

test case:
PS C:\Users\bjornr> scala -toolcp "test blank/myJar.jar" "test blank/myJar.jar"

with the suggested fix this test case will NOT print this error msg:
blank/myJar.jar""=="-toolcp" was unexpected at this time.

### CODE REVIEW

File: src/compiler/scala/tools/ant/templates/tool-windows.tmpl [hunk 1]
```diff
@@ -13,9 +13,9 @@ setlocal enableextensions enabledelayedexpansion
 set _LINE_TOOLCP=
 
 :another_param
-
-if "%1%"=="-toolcp" (
-    set _LINE_TOOLCP=%2%
+rem fix ~ according to http://ss64.com/nt/syntax-args.html
+if "%~1"=="-toolcp" (
```
Comment (retronym) [inline src/compiler/scala/tools/ant/templates/tool-windows.tmpl:17]: In your test case (`scala -toolcp "test blank/myJar.jar" "test blank/myJar.jar"`), is this change needed? AFAIS, `%1%` would be `-toolcp`, and would not contain spaces.

However, I can see how `scala "test blank/foo.scala"` _would_ need this protection. Could you please show how that works with your change?

Comment (bjornregnell) [inline src/compiler/scala/tools/ant/templates/tool-windows.tmpl:17]: Yes there are different alternatives of where quotes are needed in %~1 and %~2:
alt1: running a jar with blank in path; first param need quote handling:
scala "test blank/myJar.jar"

alt2: running a jar with blank in path for toolcp:
scala -toolcp "test blank/myJar.jar" "test blank/myJar.jar"

then the shift operation makes the nth parameter numberd as n-1 for all n...
So if AFAIK, the tilde is needed for both %1 and %2 
as any place where a path can be a parameter, space needs to be handled

**Comment by retronym** [comment]
@szeiger Could you please review this as well?

**Comment by szeiger** [comment]
LGTM. I would have expected that you need to quote "_LINE_TOOLCP=%~2" (like this) but cmd.exe seems to do the right thing regardless.

**Comment by retronym** [comment]
@bjornregnell can you please close this PR and reopen against the 2.10.x branch? Then we can include the fix in 2.10.2.

Thanks again for your contribution.

**Comment by bjornregnell** [comment]
@retronym OK. closing and reopening for 2.10.2

**Pull request closed without merging**